src/js/components/Nav/Nav.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { Box } from '../Box';
+
+const Nav = ({ ...rest }) => (
+  <Box as="nav" flex={false} gap="medium" {...rest} />
```

Comment:
What's the rationale for `gap` defaulting to `medium`? I was envisioning using this in a Sidebar with Buttons, in which case I would want gap to be undefined.

Reply:
I agree it would have made more sense to use with `items`

Reply:
So, are we going to remove the `gap="medium"` here?